src/resolvers/projectResolver.ts

```diff
@@ -610,14 +604,7 @@ export class ProjectResolver {
     query = ProjectResolver.addUserReaction(query, connectedWalletUserId, user);
     const project = await query.getOne();
 
-    if (
-      (project?.statusId === ProjStatus.drafted ||
-        project?.statusId === ProjStatus.cancelled) &&
-      // If project is draft or cancelled, just owner can view it
-      project?.admin !== String(user?.userId)
-    ) {
-      return null;
-    }
+    await canUserVisitProject(project, String(user?.userId));
```

Comment:
`canUserVisitProject` is not async
```suggestion
   canUserVisitProject(project, String(user?.userId));
```

Reply:
@amin Thanks done